contracts/Marketplace.sol

```diff
@@ -489,11 +491,11 @@ contract Marketplace is SlotReservations, Proofs, StateRetrieval, Endian {
     if (state == RequestState.New || state == RequestState.Started) {
       return end;
     } else {
-      return Math.min(end, block.timestamp - 1);
+      return Math.min(end, uint64(block.timestamp) - 1);
     }
   }
 
-  function requestExpiry(RequestId requestId) public view returns (uint256) {
+  function requestExpiry(RequestId requestId) public view returns (uint64) {
```

Comment:
Commenting there because i couldn't comment on unchanged lines...
Should we also limit the timestamp params in `_slotPayout` (there are two overloads of `_slotPayout`)?
```diff
function _slotPayout(
    RequestId requestId,
-    uint256 startingTimestamp,
-    uint256 endingTimestamp
+   uint64 startingTimestamp,
+   uint64 endingTimestamp
```